terraso_backend/apps/graphql/schema/groups.py

```diff
@@ -9,24 +10,40 @@
 from .commons import BaseDeleteMutation, BaseWriteMutation, TerrasoConnection
 
 
-class GroupNode(DjangoObjectType):
-    id = graphene.ID(source="pk", required=True)
+class GroupFilterSet(django_filters.FilterSet):
+    # TODO: members__email was kept for backward compatibility. Remove as soon
+    # as the web client be updated to use memberships__email filter
```

Comment:
question: do we have an issue tracking this? If not, please make one.

Reply:
Created:
- https://github.com/techmatters/terraso-web-client/issues/182
- https://github.com/techmatters/terraso-backend/issues/105